README.md

```diff
@@ -50,6 +50,23 @@ The reason why is that we want to focus on the creation of software and not on a
 
 We will not accept a project targeted to manipulate people, to rip people off, or put their savings at risk, such as insurance+retirement funds.
 
+## Hourly Commitment and Core Staff Membership
+
+Core staff membership is optional and conditioned on meeting committed work hours.
+Membership is required to participate in the weekly status calls, S3 meetings, and Sociocracy sessions.
+
+### Daily and Weekly Commitments
+
+- **Expected Daily Commitment**: 6 hours per working day, on average (excluding days off).
+- **Weekly Commitment**: To maintain core staff membership, you must work at least 90% of your weekly committed hours.
+
+If you fail to meet 90% of your weekly committed hours for two consecutive weeks, you will lose your core staff membership until you recover.
+Losing core staff membership does not remove you from Slack channels.
+If you lose core staff membership, you can still participate in S3 meetings by bringing a driver.
+You may also attend without a driver if necessary.
```

Comment:
```suggestion
```

Reply:
this makes just things confusing; we may ask even outsiders to participate in any meeting if necessary